File: backend/DataAccess/DaoGenerated/システム共有/tm_afmenuDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             メニューマスタ
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_afmenuDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tm_afmenu";
		public string kinoid { get; set; }                        //機能ID
		public string oyamenuid { get; set; }                     //親メニューID
		public int orderseq { get; set; }                         //並びシーケンス
		public bool paramkeisyoflg { get; set; }                  //検索パラメーター継承フラグ
		public bool addctrlflg { get; set; }                      //追加権限制御フラグ
		public bool updctrlflg { get; set; }                      //修正権限制御フラグ
		public bool delctrlflg { get; set; }                      //削除権限制御フラグ
		public bool pnousectrlflg { get; set; }                   //個人番号利用権限制御フラグ
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tm_afmenuDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             メニューマスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Db
    Public Class tm_afmenuDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_afmenu"
        Public Property kinoid As String                        '機能ID
        Public Property oyamenuid As String                     '親メニューID
        Public Property orderseq As Integer                         '並びシーケンス
        Public Property paramkeisyoflg As Boolean                  '検索パラメーター継承フラグ
        Public Property addctrlflg As Boolean                      '追加権限制御フラグ
        Public Property updctrlflg As Boolean                      '修正権限制御フラグ
        Public Property delctrlflg As Boolean                      '削除権限制御フラグ
        Public Property pnousectrlflg As Boolean                   '個人番号利用権限制御フラグ
    End Class
End Namespace
